README.md

```diff
@@ -25,79 +25,95 @@ evmiles97_00002, evmiles97, "Perfect weather for a hike"
+1. Create a new instance of `UserCollection`.
+1. Load user data from a CSV file into an instance of `UserCollection`.
+1. Create a new instance of `UserStatusCollection`.
+1. Load status data from a CSV file into an instance of `UserStatusCollection`.
+1. Perform different operations (add, update, delete, search) on the `User` and `UserStatus` collections.
 
 
 # Test-Driven Development (TDD)
 
 ## Testing
 
-Before writing any code into *main.py*, create a file called *test_main.py*, which will contain your unit tests. Your unit tests will follow these requirements:
+Before writing any code into `main.py`, create a file called `test_main.py, which will contain your unit tests. Your unit tests will follow these requirements:
 
-* You need to individually test all functions and methods within *main.py*, *users.py* and *user_status.py*.
-* Try to use *Mock()* and *Patch()* at least **once** to isolate your tests. For example, if you are testing *delete_user()* in *main.py*, your code will probably need to call *delete_user()* in *users*.py. Use *Mock()* or *Patch()* to override the response from *users.UserCollection.delete_user()*.
+* You need to individually test all functions and methods within `main.py`, `users.py` and `user_status.py`.
+* Try to use `Mock()` and `Patch()` at least **once** to isolate your tests. For example, if you are testing `delete_user()` in `main.py`, your code will probably need to call `delete_user()` in `users.py`. Use `Mock()` or `Patch()` to override the response from `users.UserCollection.delete_user()`.
 * Note that you might need multiple tests of multiple assertions on the same function or method in order to cover several scenarios.
+* You will likely want to use fixtures to set up temporary data for your tests to work with.
 
-Coverage analysis must show **100% test coverage on all files** once your code is ready to be tested.
+Coverage analysis must show **100% test coverage on all files** once your code is complete.
 
-**As you write your code inside of *main.py*, make sure you add one or more tests in *test_main.py* to target the code that you just added. In this way, your tests will be ready at the same time as the rest of your code.**
+**As you write your code inside of `main.py`, make sure you add one or more tests in `test_main.py` before you write the code itself. This is basic premise of TDD: write the tests before the actual code. In this way, your tests and the rest of your code will be developed together. And you should have 100% ceverage as soon as the code is complete**
 
 ## Additional considerations for testing
 
- Make sure you add tests for error conditions as well, including those triggering exceptions. Think what happens if:
+Make sure you add tests for error conditions as well, including those triggering exceptions. Think what happens if:
 
 * Parameters are missing.
 * Parameters are invalid (for example, a user_id value that contains spaces).
 * There are empty fields.
```

Comment:
I'm not sure what this means -- where would fields be empty? in the CSV files ??